### DISCUSSION TITLE
Error while installing the OpenROAD

### DISCUSSION THREAD

**Discussion by sreejisg** [The-OpenROAD-Project/OpenROAD-flow-scripts#1194] [General]
<img width="960" alt="image" src="https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/assets/18732533/6070c151-66c2-4942-bd17-36e4b4216acd">
The installation is stopped at this point

**Comment by luarss** [comment]
https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/issues/new?assignees=&labels=&projects=&template=bug_report_with_orfs.yml 

Could you please post your issues using this format here, will be more helpful for debugging.

**Comment by vijayank88** [comment]
@sreejisg 
Please share terminal screenshot with error message, as log not shown any error message.
